frontend/src/pages/CalendarPage/components/EventCard/index.js

```diff
@@ -10,10 +10,18 @@ import styles from './styles';
 
 import { Card, GetLocation } from '@SharedComponents';
 
-function EventCard({ event, classes }) {
-  return (
-    <>
-      {event && (
+function EventCard({ event, onClose, classes }) {
+  // Listens to whether an event is selected or not from Calendar parent comp
+  const isOpen = event ? true : false;
+
+  const ref = React.createRef();
+
+  const ModalContent = React.forwardRef(() => (
```

Comment:
I'm just wondering why you are using createRef() and forwardRef() here but in forwardRef() you don't use the ref being passed down. Any reason as to this?

Otherwise, everything else looks good.

Reply:
I was getting an error for below that the Modal type needed a reference and an inner forwardRef or something, was not completely sure how to fix it but this did the job. Should I also pass in the ref to ModalContent (forwardRef) too?

Reply:
If you're not using it then no need. Yeah if this makes it work then that's fine, but document why you chose to use it w/ a comment or something, and then we can probably merge this in